Source/Orts.Simulation/Orts.Simulation.csproj

```diff
@@ -42,6 +44,9 @@
       <HintPath>..\3rdPartyLibs\GNU.Gettext.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.Xna.Framework, Version=3.1.0.0, Culture=neutral, PublicKeyToken=6d5c3888ef60e27d, processorArchitecture=x86" />
+    <Reference Include="Microsoft.CodeDom.Providers.DotNetCompilerPlatform, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.3.5.0-preview1\lib\net45\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.dll</HintPath>
```

Comment:
CodeDom 3.6.0 is out and it's a non-preview version, so can we bump this package up to there?

Reply:
Alright, it has been updated.

By the way, my title is wrong : we are in fact supporting C# v8. ;)